### PR TITLE
Replace attributes with CSS for w3c validation

### PR DESCRIPTION
**Pull request by nitriques**
Most of the attributes are now deprecated, it is safe to remove them

### CODE REVIEW

File: workspace/utilities/lib/iframe-copyright-288.xsl [hunk 1]
```diff
@@ -59,13 +59,10 @@
...
-			<add margin="0" />
-			<add border="0" />
-			<add scrolling="no" />
-			<add frameborder="no" />
-			<add width="{$w}" />
-			<add height="{number($h)}" />
+			<set src="{$url}" />
+			<set width="{$w}" />
+			<set height="{number($h)}" />
+			<set style="border:0;margin:0;overflow:hidden">
```
Comment (PascalPiche) [inline workspace/utilities/lib/iframe-copyright-288.xsl]: missing ; at the end.
We should put space between each property to allow simple partial alteration with rem later.

Comment (nitriques) [reply workspace/utilities/lib/iframe-copyright-288.xsl]: I do not want it to be removed. the extra ; is not needed.

Comment (PascalPiche) [reply workspace/utilities/lib/iframe-copyright-288.xsl]: the extra; is needed if someone add a style attribute it will break.

Comment (nitriques) [reply workspace/utilities/lib/iframe-copyright-288.xsl]: done in 3c2f3f1

**Comment by nitriques** [comment]
@PascalPiche Is it ok with you ?

**Pull request merged**